docs/project/changelog.md

```diff
@@ -15,6 +15,9 @@ myst:
 
 ## Unreleased
 
+- {{ Update }} Pyodide now runs Python 3.11.2.
```

Comment:
Can you combine this with the changelog for Pyodide runs Python 3.11.1? Just change the version and add the second pr xref.

Reply:
Done and pushed.